media/js/src/Asset.js

```diff
@@ -22,6 +24,17 @@ export default class Asset {
             return null;
         }
 
+        // Use youtube's thumbnail format for getting the biggest
+        // thumbnail available.
+        if (
+            this.asset.primary_type === 'youtube' &&
+                this.asset.sources &&
+                this.asset.sources.url
+        ) {
+            const youtubeId = getYouTubeID(this.asset.sources.url.url);
```

Comment:
Is this double `url.url` correct?

Reply:
Yeah, the asset `sources` array has a `url` object:

![out](https://user-images.githubusercontent.com/59292/72100081-f52a8b00-32ef-11ea-9038-1e7bf4173a44.png)

On second glance, maybe I should use youtube.url here? In this case, they both contain the youtube ID so it doesn't matter.